interni/norme-di-progetto/components/processi-di-supporto/accertamento-della-qualita.tex

```diff
@@ -85,4 +70,54 @@ \subsubsection{Attività}%
   \caption{ciclo di Deming o PDCA.}
 \end{figure}
 
+\subsubsection{Metriche}%
+\label{subs:accertamento_della_qualita/metriche}
+
+%\paragraph{Denominazione delle metriche}%
+%\label{par:denominazione_delle_metriche}
+
+Le metriche sono essenziali per avere una misura oggettiva di qualità del nostro prodotto, e vengono definite durante la pianificazione del processo di accertamento della qualità.
+Per tale ragione per la definizione delle metriche di qualità è necessario identificare una denominazione comune.
+Ogni metrica avrà la seguente struttura:
+\begin{center}
+  \textbf{M[Tipo][Sigla]}
+\end{center}
+\begin{description}
+  \item [Tipo] indica il tipo di metrica. Può essere:
+        \begin{description}
+          \item[PS]: metrica di processo.
+          \item[PR]: metrica di prodotto.
+          \item[TS]: metrica di test.
+        \end{description}
```

Comment:
Il punto in fondo ad ogni item è necessario?

Reply:
mmm mi pare di no, se non sbaglio il punto andava solo nell'ultima riga se era un elenco di frasi semplici o tecnologie ecc, tipo questo, invece andava in tutte se erano frasi complesse, quidni soggetto verbo complemento tipo

Reply:
Sì è così, era per chiedere se erano stati messi per un preciso motivo oppure no